components/combobox/index.css

```
@@ -213,6 +220,27 @@
 	.spectrum-Popover.is-open {
 		transform: translateY(var(--mod-combobox-spacing-edge-to-menu, var(--spectrum-combobox-spacing-edge-to-menu)));
 	}
+
+	&.is-readOnly .spectrum-Combobox-textfield {
```

I'm wondering if targeting the `spectrum-Combobox-textfield` is the right thing here...I tried a little bit to recreate some of the other CSS I've seen with `:not()`. I was thinking the "not quiet" part would be helpful up here too, that way we could really make sure the quiet styles are separated, like you have them. It feels to me like we're getting straight to the `spectrum-Combobox-input`, instead of aiming for its parent first, then the `input`. 

```suggestion
&.is-readOnly:not(.spectrum-Combobox--quiet) {
  .spectrum-Combobox-input:read-only {
    // probably the background-color and border and hover stuff in here?
  }
}
```

Let me know what you think of that. We'd want to make the changes across all 3 of these nested selectors- `default`, `is-invalid`, and `is-disabled`.